File: app/src/main/java/net/sourcefusion/agiletools/sprintcalendar/HolidayProvider.java
package net.sourcefusion.agiletools.sprintcalendar;

import org.joda.time.LocalDate;

public interface HolidayProvider {

    boolean isWeekend(LocalDate date);
    boolean isHoliday(LocalDate time);
    boolean isWorkingDay(LocalDate localDate);
}

File: app/src/main/kotlin/net/sourcefusion/agiletools/sprintcalendar/HolidayProvider.kt
package net.sourcefusion.agiletools.sprintcalendar

import org.joda.time.LocalDate

interface HolidayProvider {

    fun isWeekend(date: LocalDate): Boolean
    fun isHoliday(time: LocalDate): Boolean
    fun isWorkingDay(localDate: LocalDate): Boolean
}
